public/index.html

```diff
@@ -5,9 +5,10 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <meta property="og:image" content="%PUBLIC_URL%/og-image.png" />
```

Comment:
For now, let's keep this picture for later change if necessary

Reply:
ok. thanks RADEK!